Sources/CartonCLI/Commands/Test.swift

```diff
@@ -48,7 +53,8 @@ struct Test: AsyncParsableCommand {
   @Argument(help: "The list of test cases to run in the test suite.")
   var testCases = [String]()
 
-  @Option(help: "Environment used to run the tests, either a browser, or command-line Wasm host.")
+  @Option(
+    help: "Environment used to run the tests: \(Environment.allCasesNames.joined(separator: ", "))")
```

Comment:
@AntonioCandinho would you mind pushing this small update? I don't think I have a permission to do that myself. Otherwise it looks ready for merging to me.

Reply:
Done! Thnks 🙏